src/invoices/InvoiceForm/InvoiceForm.test.js

```diff
@@ -11,8 +11,8 @@ jest.mock('../AdjustmentsForm', () => {
   return () => <span>AdjustmentsForm</span>;
 });
 
-jest.mock('./CurrentExchangeRate', () => {
-  return () => <span>CurrentExchangeRate</span>;
+jest.mock('@folio/stripes-acq-components/lib/CurrencyExchangeRateFields/CurrencyExchangeRateFields', () => {
```

Comment:
Is there a problem to include it in the test? It might be useful from integration testing perspective